app/v1/models/request/BirthDateVerification.scala

```diff
@@ -16,22 +16,22 @@
 
 package v1.models.request
 
-import play.api.Logger
+import play.api.{Logger, Logging}
```

Comment:
Why do we need both? the Logger should be replace with Logging?

Reply:
Fixed now.